streampark-flink/streampark-flink-kubernetes-v2/streampark-flink-kubernetes-core/src/main/scala/org/apache/streampark/flink/kubernetes/v2/operator/CROperator.scala

```diff
@@ -58,6 +60,7 @@ object CROperator extends CROperator {
   /** Apply FlinkDeployment CR. */
   // noinspection DuplicatedCode
   override def applyDeployment(spec: FlinkDeploymentDef): IO[Throwable, Unit] = {
+    validateDeployFlinkDeploymentCRD
```

Comment:
`validateDeployFlinkDeploymentCRD` return an IO structure that will not be executed here.